datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -20,7 +20,7 @@ kafka-python==1.4.4; sys_platform != 'win32'
 kazoo==2.6.1; sys_platform != 'win32'
 kubernetes==8.0.1
 ldap3==2.5
-lxml==4.2.6
+lxml==4.4.1
```

Comment:
Let's bump in a separate PR to ease QA and changelog generation

Reply:
ok